test/lib/browser/commands/assert-view/errors/no-ref-image-error.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const AssertViewError = require('lib/browser/commands/assert-view/errors/assert-view-error');
+const NoRefImageError = require('lib/browser/commands/assert-view/errors/no-ref-image-error');
+
+describe('NoRefImageError', () => {
+    it('should be instance of "AssertViewError"', () => {
+        const error = new NoRefImageError();
+
+        assert.instanceOf(error, AssertViewError);
+    });
+
+    it('should contain in message info about state name and path to reference image', () => {
```

Comment:
`should content in message` - wrong!
`it should contain info about state name and path to reference image in message`

Reply:
idite v dzhopu, umniks :)